app/javascript/packages/document-capture/context/analytics.jsx

```diff
@@ -14,12 +14,12 @@ import { createContext } from 'react';
 /**
  * @typedef AnalyticsContext
  *
- * @prop {TrackEvent} addPageAction Log an action with optional payload.
+ * @prop {TrackEvent} trackEvent Log an action with optional payload.
```

Comment:
should we link to the controller here with the list of payload names it recognizes?

Reply:
Technically the controller will handle any event we throw at it, and the allowlist only exists to avoid the "Frontend:" prefixing. But I think it wouldn't hurt to include a reference to it.

Reply:
Actually, since this context is specific to document capture, it'd probably be more applicable in the `@18f/identity-analytics` package, and it might be something we could surface in a `README.md` there, which would be good to have anyways. I think I'll go that route.

Reply:
> it might be something we could surface in a `README.md` there, which would be good to have anyways. I think I'll go that route.

Added in be566e7dcd95ba07c7856bcf0d26666dee370885.